crates/sui-graphql-e2e-tests/tests/transactions/programmable.exp

```diff
@@ -163,7 +163,7 @@ Response: {
           "effects": {
             "status": "SUCCESS",
             "errors": null,
-            "lamportVersion": 2,
+            "lamportVersion": 3,
```

Comment:
This test highlights the bug where the GraphQL implementation was previously treating the output version of package as the lamport version of the transaction.